africanus/calibration/utils/compute_and_corrupt_vis.py

```diff
@@ -0,0 +1,153 @@
+# -*- coding: utf-8 -*-
+
+import numpy as np
+from africanus.util.docs import DocstringTemplate
+from africanus.util.numba import generated_jit, njit
+from .utils import check_type
+from africanus.constants import minus_two_pi_over_c as m2pioc
+
+DIAG_DIAG = 0
+DIAG = 1
+FULL = 2
```

Comment:
These constants seem to be defined in multiple locations. It would be better if they were placed in a single python module and imported into other locations in the code. Perhaps `africanus.calibration.strategies`?. Or `africanus.constants`

Reply:
I'll put them in utils and import from there since that is where the function lives that checks for them